tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -27,9 +27,17 @@ def _test_ios_packages(args):
     if not c_framework_dir.is_dir():
         raise FileNotFoundError('c_framework_dir {} is not a folder.'.format(c_framework_dir))
 
-    framework_path = os.path.join(c_framework_dir, 'onnxruntime.framework')
-    if not pathlib.Path(framework_path).exists():
-        raise FileNotFoundError('{} does not have onnxruntime.framework'.format(c_framework_dir))
+    has_framework = pathlib.Path(os.path.join(c_framework_dir, 'onnxruntime.framework')).exists()
```

Comment:
>has_framework [](http://example.com/codeflow?start=4&length=13)

why do we still want to support the old style here?

Reply:
For now I don't think cmake can generate xcframework by itself, will still keep the framework for individual ABI, since we will also need it go generate xcframework